index.html

```diff
@@ -47,6 +47,157 @@ <h3>Let's CONNECT</h3>
+                <div class="work-details">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-skills">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-button"><button>See Project</button></div>
+            </div>
+            <div class="work-container">
+                <div class="port-image"><img src="assets/work4.png" alt=""></div>
+                <div class="work-title"><h1>Multi-Post Stories</h1></div>
+                <div class="work-technology">
+                    <ul>
+                        <li class="">CANOPY</li>
+                        <li class=""><img src="assets/counter.png" alt=""></li>
+                        <li class="tech">Back End Dev</li>
+                        <li class=""><img src="assets/Counter.png" alt=""></li>
+                        <li class="date">2015</li>
+                    </ul>
+                </div>
+                <div class="work-details">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-skills">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-button"><button>See Project</button></div>
+            </div>
+        </section>
+
+        <section class="about">
+            <div class="about-container">
+                <div class="personal-info-title">
+              <h1><a href="#" class="header-info about-header">About MySelf</a></h1>
+              </div>
+              <div class="about-info">
+              <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+              </div>
+              <div class="personal-social-info">
+                <h3>Let's CONNECT</h3>
+                <div class="social-media">
+                    <ul>
+                    <li><a href="#"><img src="assets/twitter.svg" alt="twitter"></a></li>
+                    <li><a href="#"><img src="assets/in.png" alt="linkdin"></a></li>
+                    <li><a href="#"><img src="assets/m.svg" alt=""></a></li>
+                    <li><a href="#"><img src="assets/git.svg" alt="github"></a></li>
+                    <li><a href="#"><img src="assets/peace.svg" alt="peace"></a></li>
+                </ul>
+                </div>
+                  <div class="work-button">
+                      <button>Get my resume</button>
```

Comment:
- [ ] Kindly  follow the button styling for when it is  Active, Hover and Pressed check screenshot below ⬇️ 
![btn](https://user-images.githubusercontent.com/69638013/159018519-c77dc5dc-3b4c-475d-9344-ee807f96bddc.png)